### ISSUE TITLE
Issues with interactive-image-buider "typing"

### ISSUE THREAD

**Issue by strangiato** [opendatahub-io-contrib/workbench-images#31]
When working with the interactive-image-builder if you attempt to type a number before the script finishing "typing" the number you type will not appear in the UI, but it appears to still register it.  So if you attempt to type the number it throws an error because it thinks you provided an invalid option:

![image](https://github.com/opendatahub-io-contrib/workbench-images/assets/1628206/a66636a5-e3b3-424a-82a1-a8d1882aa36f)

**Comment by guimou** [comment]
Yeah, I know. What I don't know how to do is how to prevent keys typing from register while the text is displayed.
I can always disable the type-writer effect, but that's cool... :smile: 
People, please upvote or downvote this comment if you want me to keep or remove the effect. Which you can do by yourself anyway by editing the last function in `helper-functions` (simply set delay to 0).

**Comment by shalberd** [comment]
> upvote or downvote this comment if you want me to keep or remove the effect.

keep type-writer effect: thumbs up on Guillome's comment
remove type-writer effect: thumbs down on Guillome's comment

I started. And you are right, it is overall a minor issue, can be fixed manually, yet potentially confusing users
